presto-benchmark-runner/src/test/java/com/facebook/presto/benchmark/BenchmarkTestUtil.java

```diff
@@ -0,0 +1,154 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark;
+
+import com.facebook.presto.benchmark.framework.BenchmarkQuery;
+import com.facebook.presto.benchmark.framework.BenchmarkSuite;
+import com.facebook.presto.benchmark.framework.BenchmarkSuiteInfo;
+import com.facebook.presto.benchmark.framework.PhaseSpecification;
+import com.facebook.presto.testing.mysql.MySqlOptions;
+import com.facebook.presto.testing.mysql.TestingMySqlServer;
+import com.google.common.collect.ImmutableList;
+import io.airlift.units.Duration;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.sqlobject.SqlObjectPlugin;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class BenchmarkTestUtil
+{
+    public static final String CATALOG = "benchmark";
+    public static final String SCHEMA = "default";
+    public static final String XDB = "presto";
+    public static final String BENCHMARK_SUITES_TABLE = "benchmark_suites";
+    public static final String BENCHMARK_QUERIES_TABLE = "benchmark_queries";
+
+    private static final MySqlOptions MY_SQL_OPTIONS = MySqlOptions.builder()
+            .setCommandTimeout(new Duration(90, SECONDS))
+            .build();
+
+    private BenchmarkTestUtil()
+    {
+    }
+
+    public static TestingMySqlServer setupMySql()
+            throws Exception
+    {
+        TestingMySqlServer mySqlServer = new TestingMySqlServer("testuser", "testpass", ImmutableList.of(XDB), MY_SQL_OPTIONS);
+        try (Handle handle = getJdbi(mySqlServer).open()) {
+            handle.execute("CREATE TABLE benchmark_suites (\n" +
+                    "  id bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n" +
+                    "  suite varchar(256) NOT NULL,\n" +
+                    "  query_set varchar(256) NOT NULL,\n" +
+                    "  phases mediumtext NOT NULL,\n" +
+                    "  session_properties mediumtext NOT NULL,\n" +
+                    "  created_by varchar(256) NOT NULL,\n" +
+                    "  create_time timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,\n" +
+                    "  update_time timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,\n" +
+                    "  PRIMARY KEY (id),\n" +
+                    "  UNIQUE KEY suite (suite))");
+
+            handle.execute("CREATE TABLE benchmark_queries (\n" +
+                    "  id bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n" +
+                    "  query_set varchar(256) NOT NULL,\n" +
+                    "  name varchar(256) NOT NULL,\n" +
+                    "  catalog varchar(256) NOT NULL,\n" +
+                    "  benchmark_schema varchar(256) NOT NULL,\n" +
```

Comment:
This should be called `schema`. Use back quote to avoid being treated as reserved words. See `FunctionNamespaceDao`.